.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Generate artifacts
+
+on:
+  push:
+    paths:
+      - 'src/**.md'
+  workflow_dispatch:
+
+jobs:
+  generateFiles:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
"know how much time this will take in GitHub Actions environment" we don't care for the time, it's automated :D I guest 1 or 2min max

why does it need a specific version ? what if the version becomes deprecated ? can't it work if it uses the last python version ?

Reply:
At this time, Python binaries are installed from the file descriptor https://github.com/frkim/ApplicationAssessment/blob/d28b422a382953cdccfb2708ace7344ba278575f/.python-version. This file is read by pyenv, which selects the right Python binaries version automatically.

Changes appears between Python versions, even if not breaking (code from `3.9` can work in `3.11`), the newest functionalities are mostly incompatible with older versions (code from `3.11` may not work in `3.9`).

In our case, we are using `3.11`. Running it with a `3.10` or a `3.9`, code may not work. 

Ensure you are using the same language version results in lot less headaches, often 🙂

Reply:
I thought it was an older version. Normally, Github agents always use the last one but that's not the case for python (https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md)

then, this action is the official one to use: https://github.com/actions/setup-python (your second one :))

Reply:
added to the workflow. @frkim up to you :)